components/tooltip/index.css

```diff
@@ -14,8 +14,6 @@
 @import "@spectrum-css/commons/overlay.css";
 
 .spectrum-Tooltip {
-	--spectrum-tooltip-backgound-color-default-neutral: var(--spectrum-neutral-subdued-background-color-default);
```

Comment:
![wwe-i-sentence-you-to-deletion](https://github.com/user-attachments/assets/d136ac0d-1186-4956-adf7-a294583c8148)